src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -156,6 +183,14 @@ function useCheckEncryptionState(state: State, setState: (state: State) => void)
         if (state === "loading") checkEncryptionState();
     }, [checkEncryptionState, state]);
 
+    useTypedEventEmitter(matrixClient, ClientEvent.AccountData, (event: MatrixEvent): void => {
+        const type = event.getType();
+        // Recheck the status if this account data has been updated as this implies it has changed
+        if (type === "m.org.matrix.custom.backup_disabled") {
+            checkEncryptionState();
```

Comment:
Moving from https://github.com/element-hq/element-web/pull/29113/files#r1956093155

Possibly, but in that situation the user would be changing their encryption settings on another device at the same time - wouldn't it be worse if we didn't update and let them try to turn it on again?